src/qibolab/instruments/qm.py

```diff
@@ -585,7 +585,7 @@ def play_pulses(qmsequence, relaxation_time=0, raw_adc=False):
                     save(qmpulse.shot, qmpulse.shots)
 
     @staticmethod
-    def fetch_results(result, ro_pulses, raw_adc=False):
+    def fetch_results(result, ro_pulses, average, raw_adc=False):
```

Comment:
```suggestion
    def fetch_results(result, ro_pulses, average=True, raw_adc=False):
```
You were saying it is better to put `average=True` as default, right?

Reply:
Sorry I was wrong, currently `play` does not average so it should be `average=False` to get the same behavior. So the code should be fine now.

Reply:
Yeah, I was also meant to write the suggestion for `play` and not `fetch_results` but you understood anyways :)
Thanks for the clarification.